ProjectLighthouse.Servers.GameServer/Controllers/MessageController.cs

```diff
@@ -102,12 +105,15 @@ public async Task<IActionResult> Filter(IMailService mailService)
             return this.Ok();
         }
 
-        string filteredText = CensorHelper.FilterMessage(message);
-
         string username = await this.database.UsernameFromGameToken(token);
+        
+        string filteredText = CensorHelper.FilterMessage(message);
 
-        if (ServerConfiguration.Instance.LogChatFiltering)
-            Logger.Info($"{username}: {message} / {filteredText}", LogArea.Filter);
+        if (ServerConfiguration.Instance.LogChatFiltering && filteredText != message)
+            Logger.Info($"Censored profane word(s) from in-game text sent by {username}: \"{message}\" => \"{filteredText}\"",
+                LogArea.Filter);
+        if (ServerConfiguration.Instance.LogChatMessages)
+            Logger.Info($"{username}: \"{message}\"", LogArea.Filter);
```

Comment:
This should probably be above the FilterMessage call because
1. It doesn't use filteredText
2. If the FilterMessage call fails and throws an error it will never be logged and it would be harder to track down the root cause.